### ISSUE TITLE
Guild Charter not working

### ISSUE THREAD

**Issue by Lufkins** [dalaranwow/dalaran-wow#5930]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who) Bought guild charter and its now in my bags but cannot mouse over or use

**Expected behaviour**: Tell us what should happen instead: I should have been able to get signatures from friends to create guild

**Steps to reproduce the problem**:

1. bought charter
2. Found charter in bags
3. Nothing

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** Cornhusker

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lufkins** [comment]
trying to start a guild with some real life friends, bought the charter but cannot open it.
 its in my bags on cornhusker

**Comment by RandomXD** [comment]
same for arena charter, cant make guilds, cant make arena teams, this is happening for months and idk why its being ignored

**Comment by Golganeth** [comment]
Some players are experiencing this issue and it is being addressed. For the time being, if you experience this issue make a ticket and we will help you asap.
